modin/core/io/text/text_file_dispatcher.py

```diff
@@ -642,6 +642,10 @@ def check_parameters_support(
                 filepath_or_buffer, read_kwargs.get("storage_options")
             ):
                 return False
+        elif hasattr(filepath_or_buffer, "name"):
+            # `hasattr` check is needed for the case when modin can speed up reading from buffer
+            if not cls.file_exists(filepath_or_buffer.name):
+                return False
```

Comment:
I wonder how do we support this particular speed up case - do we open it separately on workers?.. what if incoming `filepath_or_buffer` was sought to certain point before passing it in?..

Also this is unrelated to PR title I think.

Reply:
This case is not accelerated in any way, it just defaults to pandas.

Reply:
I'm speaking about the case when `filepath_or_buffer.name` _does_ exist, but the user has given us the buffer with non-zero starting position.

Reply:
oops :) this looks like a bug